package.json

```diff
@@ -9,6 +12,7 @@
     "start": "node ./scripts/run-demo.js",
     "updatePackageJson": "node ./scripts/edit-package-json.js",
     "run-demo": "node ./scripts/run-demo.js",
+    "publish-pages": "gh-pages -d ./packages/ReactDagEditor/.static-demo",
```

Comment:
Why can't I see the hostname for our online demo here? It suppose to be our org's github page

Reply:
I'm not sure what that supposed look like. We can change that after enable the github pages.